test/hotspot/jtreg/compiler/cha/cha_control.txt

```diff
@@ -0,0 +1,30 @@
+[
```

Comment:
Currently, the prevalent way to specify compiler directives is through WhiteBox API at runtime (through `WhiteBox.addCompilerDirective(String directive)`). Please, follow the same pattern here. I find it more convenient to reason about test logic when all the pieces are present in a single place.

Reply:
Thanks for your suggestion. I will change the test case.

Reply:
Thanks, the test case looks good.